js/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "md4w",
+  "description": "A WebAssembly port of md4c - a Markdown parser written in C.",
+  "version": "0.0.2",
+  "type": "module",
+  "module": "index.js",
+  "main": "index.js",
+  "exports": {
+    ".": {
+      "workerd": "./unwasm.js",
```

Comment:
This probably is not a safe assumption. Because worked loads `.wasm` as a Module instance (rather than esm exports!) if you want to directly support cloudflare target without unwasm, probably need a new entry.

Also I would put `unwasm` before `workerd` (order matters in `exports` field) this way for unwasm enabled situations, we allow bundler to optimize as much as it can

Reply:
> Also I would put unwasm before workerd (order matters in exports field) this way for unwasm enabled situations, we allow bundler to optimize as much as it can

makes sense, i will update